### ISSUE TITLE
Cant get into any rom 

### ISSUE THREAD

**Issue by Blane17** [iGBAEmu/EclipseIssues#310]
# Description
*Please describe your issue here.*

![39074790-0764-42AD-BCCC-EF801A4D6174](https://user-images.githubusercontent.com/82622098/114954798-e8fd7980-9e20-11eb-8a0e-06dae230a11b.png)

# Device Information
## States
| State | Value | 
|-------|-------| 
| Audio Enabled | true | 
| Auto-Save Rate| 30000 | 
| Current Skin Set | true | 
| Desktop Mode Enabled | false | 
| Full Screen Enabled | true | 
| Games Setup | true | 
| Repos Setup | false | 
| Skins Setup | true | 
| Eclipse Version | 2.2.0 | 
## Device
| State | Value | 
|-------|-------| 
| Device (F7) | iOSiPhoneiPhone X | 
| User Agent | Mozilla/5.0 (iPhone- CPU iPhone OS 14_4_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.3 Mobile/15E148 Safari/604.1 | 
| Do Not Track | Not Set | 
| Cookies Enabled | true | 
| ServiceWorker Support | true | 
| Languages | en-us | 
| Vendor | Apple Computer, Inc. | 
 | Platform | iPhone |

**Comment by kolsky132** [comment]
Same issue. Is there ever any solution to these issues we submit?

**Comment by Blane17** [comment]
I don’t think so

**Comment by magnetardev** [comment]
Our download proxy was down, sorry about that.

As for why we don't respond to issues here: 90% of the time the issues are things we know about or literally just empty. That, and we're working on a major update to resolve most people's issues with Eclipse, so we'd rather just address in bulk. 

tl;dr: It's not that we don't see them, it's just we don't always want to answer the same question for the 50th time.
